Source/Lib/Codec/EbEncDecProcess.c

```diff
@@ -3221,12 +3222,11 @@ void* EncDecKernel(void *inputPtr)
                     EbReleaseObject(pictureControlSetPtr->refPicPtrArray[1]);
                 }
             }
-        }
-        EbReleaseObject(encDecTasksPtr->pictureControlSetWrapperPtr);
-
 #if DEADLOCK_DEBUG
-        SVT_LOG("POC %lld ENCDEC OUT \n", pictureControlSetPtr->pictureNumber);
+            SVT_LOG("POC %lu ENCDEC OUT \n", pictureControlSetPtr->pictureNumber);
 #endif
```

Comment:
I run this PR, the line sometimes shows later than `PK OUT`.

Reply:
This makes sense, due to the race condition that, PK triggered from Line 3051 may happen before the current picture completes its encoding with SAO and other stuff.